.azure-pipelines-templates/matrix.yml

```diff
@@ -119,17 +146,28 @@ jobs:
 
   # Publish release
```

Comment:
Slightly confusing comment since it covers several jobs, I think its really `Build, test, and publish a release`?